src/components/ebay-fake-menu/examples/03-fake-item-disabled/template.marko

```diff
@@ -0,0 +1,5 @@
+<ebay-fake-menu>
+    <@item href="#" current>item 1</@item>
```

Comment:
Wondering if we should just remove a layer of abstraction here, and have `<@link>` and `<@button>` instead of `<@item>`?

Reply:
I was thinking the same thing actually but ended up not doing it because it might be a lot of changes.
But since you brought it up, I'll go ahead and take care of it.

Reply:
And the code would still have to change them to `<@item type="button or link">` so we would still support both formats.

Maybe we can do this later since it won't be a breaking change.

Reply:
Yep, let's save for later